random-distribution-helpers.go

```diff
@@ -14,60 +14,66 @@ type RandomDistribution interface {
 	String() string
 }
 
-var _ RandomDistribution = RandomConstDistribution{}
+var _ RandomDistribution = &RandomConstDistribution{}
 
 type RandomConstDistribution struct {
```

Comment:
Since this always returns the same value, shouldn't it just be called ConstDistribution?
IMO, the interface should probably also be called Distribution instead of RandomDistribution? But even if that's not the case, the constant one can be called ConstDistribution, even though it implements an interface that starts with "Random".

Reply:
Distribution can be random while outputting one value. It is a distribution that has the probability of 1 for this value. RandomConstDistribution is a valid name

Reply:
Ok, but is the "random" really necessary? Isn't the statistical term just "distribution"? At least the titles on Wikipedia are all just "distribution" and "Verteilung", e.g. "continuous distribution", not "continuous random distribution".

Reply:
Does not matter. Definitions alter (e.g. http://mathworld.wolfram.com/RandomDistribution.html ) but can adjust in order to shorten the name naming